src/clockkit.cs

```diff
@@ -53,16 +53,20 @@ interface CLKComplicationDataSource {
 		[Export ("getTimelineEntriesForComplication:afterDate:limit:withHandler:")]
 		void GetTimelineEntriesAfterDate (CLKComplication complication, NSDate afterDate, nuint limit, Action<CLKComplicationTimelineEntry []> handler);
 	
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use WKRefreshBackgroundTask instead")]
 		[Export ("getNextRequestedUpdateDateWithHandler:")]
 		void GetNextRequestedUpdateDate (Action<NSDate> handler);
 
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use `WKRefreshBackgroundTask` instead")]
```

Comment:
https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#rule-2 please use apostrophes not grave accent (and add a dot at the end) (: